LayoutTests/platform/mac/accessibility/mac/aria-notify-with-options-expected.txt

```diff
@@ -0,0 +1,31 @@
+Tests that ariaNotify sends the right userInfo when options are specified.
+
+Received announcement request:
+AnnouncementKey: San Francisco
+AXARIAAnnouncementInterruptBehavior: None
+AXARIAAnnouncementPriority: High
+AXAnnouncementLangaugeKey: en
+
+Received announcement request:
+AnnouncementKey: Chicago
+AXARIAAnnouncementInterruptBehavior: All
+AXARIAAnnouncementPriority: High
+AXAnnouncementLangaugeKey: en
+
+Received announcement request:
+AnnouncementKey: Boston
+AXARIAAnnouncementInterruptBehavior: Pending
+AXARIAAnnouncementPriority: High
+AXAnnouncementLangaugeKey: da
+
+Received announcement request:
+AnnouncementKey: Boston
+AXARIAAnnouncementInterruptBehavior: None
+AXARIAAnnouncementPriority: High
+AXAnnouncementLangaugeKey: da
```

Comment:
Language is misspelled here; please check for the misspelling throughout the patch